pkg/kubelet/dockershim/docker_image.go

```diff
@@ -33,7 +34,8 @@ import (
 func (ds *dockerService) ListImages(filter *runtimeapi.ImageFilter) ([]*runtimeapi.Image, error) {
 	opts := dockertypes.ImageListOptions{}
 	if filter != nil {
-		if imgSpec := filter.GetImage(); imgSpec != nil {
+		if imgSpec := filter.GetImage(); imgSpec != nil && imgSpec.Image != "" {
```

Comment:
Should we just use filter.GetImage().GetImage() instead?

Reply:
Good catch. Updated, PTAL